tools/lint-all

```diff
@@ -458,8 +458,9 @@ def run():
             sys.exit(1)
 
     by_lang = cast(Dict[str, List[str]], lister.list_files(args, modified_only=options.modified,
```

Comment:
Putting `lister.list_files()` on a separate line could make the arguments easier to read.